regulations/templates/regulations/section.html

```diff
@@ -19,13 +19,13 @@
 
 <div class="box">
 
-    <aside class="left-sidebar" data-state="expanded">
+    <aside class="left-sidebar" data-state="expanded" data-target="left-sidebar">
```

Comment:
keep the namespace in some way, `data-target` leaves open the question "target for what?"

Reply:
Resolved in 60cdf72